src/common/getters.ts

```diff
@@ -50,20 +56,22 @@ export function getResources(
  * @example
  *
  * ```
- * const ownerWebId = getResourceOwner(gConsentGrant);
+ * const ownerWebId = getResourceOwner(accessGrant);
  * ```
  *
  * @param vc The Access Grant/Request
  * @returns The resource owner WebID
  */
-export function getResourceOwner(vc: AccessGrantGConsent): string;
 export function getResourceOwner(
-  vc: AccessGrantGConsent | AccessRequestGConsent
+  vc: AccessGrantGConsent | AccessGrantOdrl
+): string;
+export function getResourceOwner(
+  vc: AccessGrantGConsent | AccessRequestGConsent | AccessGrantOdrl
```

Comment:
I think I screwed up the overload initially, because I don't think the signature
```
export function getResourceOwner(vc: AccessGrantGConsent): string;
export function getResourceOwner(
  vc: AccessGrantGConsent | AccessRequestGConsent
): string | undefined;
export function getResourceOwner(
  vc: AccessGrantGConsent | AccessRequestGConsent
): string | undefined
```
makes sense, it should rather be 
```
export function getResourceOwner(vc: AccessGrantGConsent): string;
// No longer a type disjunction here
export function getResourceOwner(vc: AccessRequestGConsent): string | undefined;
export function getResourceOwner(
  vc: AccessGrantGConsent | AccessRequestGConsent
): string | undefined
```
and so that means the new signature would look like 
```
export function getResourceOwner(vc: AccessGrantGConsent | AccessGrantOdrl): string;
export function getResourceOwner(vc: AccessRequestGConsent): string | undefined;
export function getResourceOwner(
  vc: AccessGrantGConsent | AccessRequestGConsent | AccessGrantOdrl
): string | undefined
```

Reply:
Why would this not return `string | undefined`?
``` 
export function getResourceOwner(vc: AccessGrantGConsent | AccessGrantOdrl): string;
```

Reply:
A Grant being issued by the resource owner, it is mandatory that the VC subject is the resource owner. The resource owner may not be defined in an Access Request, because a Request may be scoped to a target resource with access modes, without specifying who should grant access.

Reply:
My bad, I assumed the disjunction would be picked up from the overload, which isn't correct, so @VirginiaBalseiro you were right.